release-tools/check-crate-version/toml-diff/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "toml-diff"
+version = "0.0.0"
```

Comment:
Can you also add `publish = false`? I don't think we intend to publish this crate at this time. At least not while it is living in this repo

Reply:
Fixed